Source/WebKit/NetworkProcess/storage/FileSystemStorageManager.cpp

```diff
@@ -90,6 +91,9 @@ Expected<WebCore::FileSystemHandleIdentifier, FileSystemStorageError> FileSystem
         }
     }
 
+    if (!fileExists && createIfNecessary)
```

Comment:
I don't quite understand this change. File System API does not directly work store blob, and this patch does not validate that file paths are accessible in subsequent File System requests. 
(Also subsequent requests only contain identifier, so I don't understand how this can be useful...)

Reply:
At one point I thought I needed this change to make `FileSystemDirectoryHandle.getFileHandle(..., {create = true})` work properly, but I don't think it's needed anymore. I've removed it and made the other changes requested.

Reply:
Actually I think this is still needed as tests like mentioned above (ex. imported/w3c/web-platform-tests/fs/FileSystemDirectoryHandle-getFileHandle.https.any.html ) will not work without this. It wasn't showing up before due to an error piping through the feature flag. I'm going to reincorporate this.

Reply:
Checking the test, there is indeed one case where a file-backed blob can be created from path returned File System API: `FileSystemFileHandle::getFile()`. 
We can make network process adds the path to allowlist before replying the message. i.e. Make `NetworkStorageManager::getFile()` dispatch a task to main thread and ask `m_process` to add the path for the connection. Note `void NetworkStorageManager::getFile` should take `IPC::Connection&` as it first parameter.